### PR TITLE
Pazuz machine pairing

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Integrate hedging and accounting with the rest of the services](https://trello.com/c/uKaku0j4/973-integrate-hedging-and-accounting-with-the-rest-of-the-services)

File: lib/new-admin/middlewares/cleanUserSessions.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const schemaCache = {}
+
+const cleanUserSessions = (cleanInterval) => (req, res, next) => {
+  const schema = asyncLocalStorage.getStore() ? asyncLocalStorage.getStore().get('schema') : null
+  const now = Date.now()
+
+  if (!schema) return next()
+  if (schema && schemaCache.schema + cleanInterval > now) return next()
+
+  console.log('Clearing expired sessions for schema', schema)
```
Comment (josepfo) [inline lib/new-admin/middlewares/cleanUserSessions.js:14]: Remove this

File: new-lamassu-admin/src/pages/Authentication/Register.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import { useQuery, useMutation } from '@apollo/react-hooks'
 import { makeStyles, Grid } from '@material-ui/core'
 import Paper from '@material-ui/core/Paper'
+// import base64 from 'base-64'
```
Comment (josepfo) [inline new-lamassu-admin/src/pages/Authentication/Register.js:4]: oh, this was already removed on other PR, rebase should fix it, right?

**Review by josepfo — CHANGES_REQUESTED** [review]

File: lib/db.js [hunk 3]
```diff
@@ -36,7 +37,10 @@ const _task = (obj, opts, cb) => {
   })
 }
 
-const getSchema = () => 'public'
+const getSchema = () => {
+  const store = asyncLocalStorage.getStore() ?? defaultStore()
+  return asyncLocalStorage.run(store, () => store.get('schema'))
```
Comment (josepfo) [inline lib/db.js:42]: shouldn't we use set() instead of get()?

**Pull request closed without merging**